example/provider-local/managedseeds/managedseedset.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: seedmanagement.gardener.cloud/v1alpha1
+kind: ManagedSeedSet
+metadata:
+  name: my-mss
+  namespace: garden # Must be garden
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: my-managed-seed-set
+  template:
+    # <See `example/55-managedseed-gardenlet.yaml` for more details>
+    metadata:
+      labels:
+        name: my-managed-seed-set
+    spec:
+      # shoot.name must not be specified
+      gardenlet:
+        config: # GardenletConfiguration resource
+          # <See `example/20-componentconfig-gardenlet.yaml` for more details>
+          apiVersion: gardenlet.config.gardener.cloud/v1alpha1
+          kind: GardenletConfiguration
+          seedConfig:
+            metadata:
+              labels:
+                name: my-managed-seed-set
+            spec:
+              dns:
+                # "replica-name" in DNS domains will be replaced by the actual replica name
+                ingressDomain: replica-name.garden.shoot.dev.k8s-hana.ondemand.com
```

Comment:
Not introduced with this PR but we should not use `shoot.dev.k8s-hana.ondemand.com` in examples and local setup manifests.

For the example manifest, we could use `replica-name.ingress.dev.my-seed.example.com`:
https://github.com/gardener/gardener/blob/2b4c4f5df0d1b7e00891964ccc53cec39e05ecd1/example/50-seed.yaml#L56

For the local setup manifest, we could use `replica-name.ingress.local.seed.local.gardener.cloud`:
https://github.com/gardener/gardener/blob/1b425822b264e319274e40b486898454cbc6ff93/dev-setup/gardenlet/base/gardenlet.yaml#L72-L73

If you find better ingress domains, feel free to use them instead of the above suggestions.